docs/src/modules/components/DiamondSponsors.js

```diff
@@ -74,9 +74,9 @@ export default function DiamondSponsors(props) {
           }}
         >
           <img
-            width="42"
+            width="150"
```

Comment:
The image is 250*70. It's not compatible with a rendering zone of 150*42

Reply:
I couldn't find the logo with the text. I will wait for @danilo-leal

Reply:
Just pushed a commit with the Octopus full logo in svg. I think it resolved the crispness of the png!